Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.py

```diff
@@ -4,13 +4,77 @@
 from enum import Enum
 from http import HTTPStatus
 from typing import Any
+from functools import partial
 
 from CommonServerPython import *
 from requests import Response
 
 LIMIT_SIZE = 50
 
 
+class ArgumentValues(Enum):
```

Comment:
Is this class meant to validate the argument values? Are all the values predefined in the yml file? If so, there is no reason to use this class and validate that the values are as they are in the yml.